tests/test_wave_01.py

```diff
@@ -143,12 +145,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][1]["title"] == HORROR_1["title"]
```

Comment:
Likewise, it would be better from a test design standpoint to be more thorough. It's even easier in this case because you can compare the entire dictionary as it has been saved in `movie_to_watch`, giving us `assert updated_data["watched"][1] == movie_to_watch`.

Though this is minor like the other test design issue.